source/git-util.js

```diff
@@ -133,6 +133,29 @@ exports.tagExistsOnRemote = async tagName => {
 	}
 };
 
+exports.defaultBranch = async options => {
+	if (options.releaseBranch) {
+		return options.releaseBranch;
+	}
```

Comment:
This utility should be kept agnostic to `np`. It's just a pure Git utility. I think it would make sense to do another utility in `util.js` that adds this check and the error on line 154.

Reply:
Or maybe we should just do this normalization on startup so `options.releaseBranch` either reflects what the user specifies or falls back to `defaultBranch`.

Reply:
I wasn't sure where to normalize. Both `np` and `cli` seemed like natural entrypoints. Let me know if you have further suggestions here.

Reply:
Maybe here: https://github.com/sindresorhus/np/blob/98172611d24af1147468f1ad86c9e425f9e40c52/source/cli-implementation.js#L125-L130 ?

Reply:
Sure! I've placed the normalization there now.

Is `np` designed to be used programmatically from Node apps? If so, the normalization won't work there (although will behave properly when used via CLI)

Reply:
> Is np designed to be used programmatically from Node apps? If so, the normalization won't work there (although will behave properly when used via CLI)

No, it's just split up like that to try to have some kind of separation of concerns.